README.md

```diff
@@ -130,6 +126,33 @@ jobs:
           pluginstxt: plugins.txt
           jcasc: jcasc.yml
 ```
+### Runtime action (Deprecated)
+This case is realized by the combination of jenkins-plugin-installation-action and jenkinsfile-runner-action. It will download all the dependencies and run the pipeline at the runtime. Its main disadvantage is the possibility of suffering from the jenkins.io outage.
```

Comment:
```suggestion
This case is realized by the combination of jenkins-plugin-installation-action and jenkinsfile-runner-action. It will download all the dependencies and run the pipeline at the runtime. Its main disadvantage is the possibility of suffering from a plugins.jenkins.io outage.
```

Reply:
Aren't all of your actions downloading plugins at runtime?

Reply:
The runtime actions also download Jenkins core, Jenkinsfile-runner and Plugin Manager. For the rest two actions, if the users don't change the plugins list, it won't download any binaries again.

Reply:
Hi Yiming. Please also see https://github.com/actions/cache#cache-limits

> A repository can have up to 10GB of caches. Once the 10GB limit is reached, older caches will be evicted based on when the cache was last accessed. Caches that are not accessed within the last week will also be evicted.

Reply:
Yeah, I know this limitation before I tried to apply it in this action. The cache is only a kind of optimization. If the users don't want to download plugins, they can build the images by extending the basic images provided by jenkinsfile-runner.

Reply:
I feel confused. Does it mean I need to delete the cache feature in these two actions?

Reply:
To be honest I am not sure if we should have the caching as default for the action if it is only a kind of optimization. At best it should be optional. This detail should be clearly stated in the docs. Maybe for consistency we should do these away for both actions for now, unless a definite need arises.

Reply:
OK, I'll make the cache optional.